src/main/java/teammates/ui/newcontroller/PostCourseEnrollSaveAction.java

```diff
@@ -77,14 +76,6 @@ public ActionResult execute() throws EntityNotFoundException {
         CourseEnrollmentResult enrollResult = logic.enrollStudents(studentsInfo, courseId);
         List<StudentAttributes> students = enrollResult.studentList;
 
-        // Adjust submissions for all feedback responses within the course
```

Comment:
Where is this operation done now?

Reply:
The operation (or scheduled task) is to cascade update the responses/comments given by a student when the `enroll` operation triggers email modification of an existing student. Since the cascade updating logic is inside `logic.enrollStudents(studentsInfo, courseId);`. There is no need to have this operation here.

However, the reason why the scheduled task is here is because cascading update of responses/comments takes time and this Action may not finish in 60s. Nevertheless, I vote for removal of the operation here:
- The cascade update only trigger when the student's email change. It is not really common in `enroll` function
- Removing of the scheduled tasks will also solve the macker suppresion as talked in https://github.com/TEAMMATES/teammates/pull/9202#discussion_r255347928 (:P this is not a reason but a bonus). I need to push an additional commit to see the benefit.
- Even if we want to keep the scheduled task due to the time constrain in GAE. The scheduled task (`FeedbackResponseAdjustmentWorkerAction`) need to be refactored (written 5 years ago :P https://github.com/TEAMMATES/teammates/blame/33485bbf5e91bfceea6a2273c34febcb717b5c79/src/main/java/teammates/logic/automated/FeedbackSubmissionAdjustmentAction.java#L23). This can be a separate issue (or I can do it just after the PR is merged).